ChegeBryan/politico
# #166179115 FetchApi delete party

#### What does this PR do?

Enables admin user to delete a party from UI

#### Description of Task to be completed?

- [x] Fetch parties from the server
- [x] Render the parties in a table
- [x] Add a dissolve party button to delete party 

#### How should this be manually tested?
After cloning the repository to your local machine - check
[README.md](README.md) for instructions. Checkout the branch `ft-fetchApi-delete-party-166179115` open the `index.html` page to serve the homepage.  Click on the sign in option use the admin logins as folows:
```json
{
    "email": "admin@politico.org",
    "password": "2019NBO37"
}
```
On the dashboard navigate to delete party option. Click on `dissolve party button` against the party you want to delete.

#### What are the relevant pivotal tracker stories?
[#166179115](https://www.pivotaltracker.com/story/show/166179115)
